### ISSUE TITLE
CSV export works only on Mac

### ISSUE THREAD

**Issue by mihaii-telenav** [maproulette/maproulette3#359]
Clicking on Export CSV does nothing on Windows. On Mac it successfully downloads the file.

**Comment by mvexel** [comment]
Can't confirm this since I don't have access to a Windows machine. From discussing with @mihaii-telenav it sounds like Chrome on Windows attempts to open a tab but fails? Can you try in Edge and Firefox as well?

**Comment by mihaii-telenav** [comment]
@mvexel That is correct. On Chrome Version 66.0.3359.139 (Official Build) (64-bit) it tries to open a new tab to download the CSV but it fails.

Weirdly enough I tried on Firefox 59.0.2 (64-bit) it works, but with a glitch. Instead of simply downloading a csv file it prompts this:

![image](https://user-images.githubusercontent.com/20751382/39907938-6476511e-54f4-11e8-985c-40f839bb264b.png)

It successfully downloads a file but it doesn't have a .csv extension:

![image](https://user-images.githubusercontent.com/20751382/39907977-899b057a-54f4-11e8-8f3f-d3979d8b2f3d.png)

You have to manually add the csv extension and only then you can open the file.

I did not try on Edge as I have a Windows 7 machine.
